tests/config/default-config-production.test.js

```diff
@@ -22,10 +23,11 @@ describe('default-config', () => {
 
   it('Returns correct values for BIOMAGE production', () => {
     const prodEnvironment = 'production';
```

Comment:
see comment above -- I think this could be one test ala `it('Return correct values for production environments')` and then loop through all keys in `domainSpecificContent`.